Framework/CurveFitting/src/FuncMinimizers/DampedGaussNewtonMinimizer.cpp

```diff
@@ -24,7 +24,7 @@ namespace {
 Kernel::Logger g_log("DampedGaussNewtonMinimizer");
 }
 
-DECLARE_FUNCMINIMIZER(DampedGaussNewtonMinimizer, Damped Gauss-Newton)
+DECLARE_FUNCMINIMIZER(DampedGaussNewtonMinimizer, Damped Gauss - Newton)
 
```

Comment:
This clang change has broken the selection of Damped Gauss-Newton in the fit properties browser.
It appears that DECLARE_FUNCMINIMIZER macro includes the quotes for its second parameter, so the second parameter does not appear to be a string. This confused me and has also confused clang. 

It looks like I can't use'-' till DECLARE_FUNCMINIMIZER is fixed.